packages/calcite-components/src/components/shell-panel/shell-panel.scss

```diff
@@ -48,21 +48,23 @@
   z-index: var(--calcite-shell-panel-z-index, calc(theme("zIndex.header") + 1));
 }
 
+:host([layout="vertical"][display-mode="float"]) .content,
+:host([layout="vertical"][display-mode="content-detached"]) .content {
+  flex: 2;
+}
+
 :host([layout="vertical"][width-scale="s"]) .content {
   --calcite-shell-panel-width-internal: var(--calcite-shell-panel-width, 12vw);
-  --calcite-shell-panel-max-width-internal: var(--calcite-shell-panel-max-width, 300px);
```

Comment:
question: Why are these max widths being removed? There's still a reference to one of them in the CSS as well.

Reply:
To make the content completely span the entire width of the shell-panel.

Reply:
This would remove the `max-width` of 300px for a shell-panel in cases where the shell panel is vertical. I think this would remove that existing default max-width of 300px.

If we do remove these, there is still a reference in the CSS to `--calcite-shell-panel-max-width-internal` that would need to be removed.